code/modules/client/preferences.dm

```diff
@@ -287,7 +287,7 @@ datum/preferences
 	else if(href_list["delete"])
 		if (!config.sql_saves)
 			return 0
-		if (alert(usr, "This will permanently delete the character you have loaded. Are you sure?", "Delete Character", "No", "Yes") == "Yes")
+		if (alert(usr, "This will permanently delete the character you have loaded. Are you sure?", "Delete Character", "Delete","Cancel") == "Delete")

```

Comment:
```suggestion
		if (alert(usr, "This will permanently delete the character you have loaded. Are you sure?", "Delete Character", "Cancel","Delete") == "Delete")
```

flip this, because I cannot be arsed to deal with people going, "i aksidentalli my character plz."